packages/nextjs/.storybook/decorators/TestHarness.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, PropsWithChildren } from "react";
+import { Decorator } from "@storybook/react";
+import { MotionConfig, MotionConfigProps } from "framer-motion";
+import { TypedParameters } from "../types";
+
+declare module "../types" {
+  export interface TypedParameters extends TestHarnessProps {}
+}
+
+export type TestHarnessProps = {
+  motionConfig?: MotionConfigProps;
+};
+
+export const TestHarness: FC<PropsWithChildren<TestHarnessProps>> = ({ motionConfig, children }) => {
+  return <MotionConfig {...motionConfig}>{children}</MotionConfig>;
+};
+
+export const TestHarnessDecorator: Decorator = (Story, ctx) => {
```

Comment:
❓ It may be a misconfiguration on my end, but I'm not seeing any types on these parameters. Is that expected?